glossary/glossary.md

```diff
@@ -120,24 +120,28 @@
     - #### **B frame**
     A B frame (Bidirectional predictive frame) is a video compression frame that relies on both the preceding and following I or P frames for data, using the motion differences between these frames to further reduce the stored information, resulting in even smaller file sizes and better compression efficiency.
 ## Membrane Framework
-- #### **Action**
-  An action can be returned from [callback](#callback) and it is a way of element interaction with other elements and parts of the framework. An exemplary action might be [`:buffer`](https://hexdocs.pm/membrane_core/Membrane.Element.Action.html#t:buffer/0) action, that sends buffers through a pad, or [`:terminate`](https://hexdocs.pm/membrane_core/Membrane.Element.Action.html#t:terminate/0) action, that terminates element with given reason.
-- #### **Callback**
-  A callback is a function defined by a user, that gets called once a particular event happens.
-  Exemplary callbacks in the Membrane Framework are: [`handle_end_of_stream/3`](https://hexdocs.pm/membrane_core/Membrane.Element.WithInputPads.html#c:handle_end_of_stream/3) being called once the end of stream event is received on some pad or [`handle_init/2`](https://hexdocs.pm/membrane_core/Membrane.Pipeline.html#c:handle_init/2) called on initialization of an [element](#element).
+- #### **Buffer**
+  A fundamental structure in Membrane that is used to send data between elements.
+- #### **Element**
+  The most basic entity that is responsible for processing multimedia. 
```

Comment:
I believe we should move the element types (source, sink etc.) description here

Reply:
Since definitions of the mentioned element types refer to concepts like `bins` or `pads` before they're introduced, I think they should stay where they are rn, with a header separating them more clearly